docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -67,11 +67,26 @@ kubestellar init
 
 (This style of deployment requires release v0.6 or later of KubeStellar.)
 
-First you will need the kcp `kubectl` plugins.  See [the "Start kcp"
-section above](../#start-kcp) for instructions on how to get all of the
-kcp executables.
-
-Next you will need to get a build of KubeStellar.  See
+You need a Kubernetes cluster with an Ingress controller deployed and
+configured in a way that does _not_ terminate TLS connections (this
+abstinence is often called "TLS passthrough"). An OpenShift cluster
+would be one qualifying thing. Another would be an ordinary Kubernetes
+cluster with the [nginx Ingress
+controller](https://docs.nginx.com/nginx-ingress-controller/) deployed
+and configured appropriately. Please note that [special
+considerations](https://kind.sigs.k8s.io/docs/user/ingress/) apply
+when deploying an ingress controller in `kind`. See [a fully worked example with kind and nginx](../../environments/dev-env/#hosting-kubestellar-in-a-kind-cluster). You will need to know
+the port number at which the Ingress controller is listening for HTTPS
+connections.
+
```

Comment:
Why does this section appears in two md files?  can we point to the section in the other file ?

Reply:
I have revised to remove the duplication.